added release notes up to 10.17.0.104 (#187)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/devices-10-17-0.md

<td>10.16.119.0</td>
</tr>

<tr>
<td>Device Management</td>
<td>When removing a device which has child devices, users can now choose either to remove a device user or to remove all child devices. Prior to this change, it was possible to choose both options in the UI, but such an action is not supported by the backend.</td>
<td>MTM-45277</td>
<td>10.17.0.103</td>
</tr>

<tr>
<td>Device Management</td>

> <td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and didn´t return any results when filtering was applied. This is now fixed.</td>

Reviewer: ```suggestion
<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and didn't return any results when a filter was applied. This is now fixed.</td>
```

<td>10.16.185.0</td>
</tr>

<tr>
<td>Loriot</td>

> <td>Events created by the Loriot agent when sending uplink requests sre now considered as device requests and provide connectivity status and availability monitoring.</td>

Reviewer: ```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now considered device requests and provide connectivity status and availability monitoring.</td>
```
